redis/src/test/scala/zio/redis/OutputSpec.scala

```diff
@@ -243,6 +243,265 @@ object OutputSpec extends BaseSpec {
             res <- Task(ChunkOptionalLongOutput.unsafeDecode(input))
           } yield assert(res)(equalTo(Chunk(Some(1L), Some(1L), Some(2L), Some(3L))))
         }
+      ),
+      suite("stream")(
+        testM("extract valid input") {
+          val input = respArrayVals(
+            respArrayVals(
```

Comment:
There are already helpers in RespValue, e.g. `array` and `bulkString`. Try to use them instead of custom ones.

Reply:
@mijicd should I remove these custom methods and switch them in other tests too?

Reply:
Preferably yes :)

Reply:
Done.